INSTALLING.md

```diff
@@ -0,0 +1,63 @@
+# Installation Options for splunk-rum.js
+
+You have a number of options for how you place the script into your pages/apps.  This document walks you through some
+of them and discusses their tradeoffs.  The best choice for you will depend on the design of your app/pages, the performance
+variables you cast most about, and how much complexity/work you want to put into your integration.
+
+
+## Default Recommendation
+
+Our recommendation for getting started is simple: host `splunk-rum.js` as your own asset
+(so that it is served over the same HTTP connection as your pages/scripts/images) and apply the following scripts in
+your `<head>`:
+
+```html
+<script src="http://your-site.com/whatever/splunk-rum.js"></script>
```

Comment:
I'd say `<script src="/whatever/splunk-rum.js"></script>`, the host part is unnecessary